salmon/common/__init__.py

```diff
@@ -24,6 +24,7 @@
     normalize_accents,
     strip_template_keys,
     truncate,
+    sizeof_fmt,
```

Comment:
wanna make this `size_of_fmt` or `fmt_size` to keep things consistent-ish maybe?

Reply:
Agreed. Went with format_size